doc/python/labels.rst

```diff
@@ -142,9 +142,9 @@ two labels refer to one another:
     # definition of 'reg'.
 
 .. note::
-    In the example above there will be no error when the label dictionary is defined.
+    In the example above, there will be no error when the label dictionary is defined.
     Instead, there will be an error later when the label dictionary is applied to
-    a morphology, and the cyclic dependency is detected when thingifying the locations
+    a morphology, and the cyclic dependency is detected when thingifying (?) the locations
```

Comment:
```suggestion
    a morphology, and the cyclic dependency is detected when thingifying the locations
```
*'Thingification'* is the term the elders agreed upon after a long and arduous fight. 
Correct would be *'reification'* (rei = greek for thing); the process of turning abstract
into --- well --- a thing. It's used in CS when you need to manipulate a concept that is normally
not an object in your language, e.g. you cannot pass a type to a (non-template) function
in C++, but you _can_ pass a numerical tag naming that type.

So, thingify is now a term in our API and thus used in the documentation. Concretely, it's used
to describe the process of mapping this `'(proximal (tag 3))'` **and** a morphology to a set of
points.

Reply:
As to why it wasn't 'reification', that was deemed to arcane for common use.